stock_cycle_count/models/stock_location.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2017-18 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+from datetime import datetime
+
+from odoo import fields, models, tools
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from statistics import mean
+
+    STATS_PATH = tools.find_in_path("statistics")
+except (ImportError, IOError) as err:
+    _logger.debug(err)
+
+
+class StockLocation(models.Model):
+    _inherit = "stock.location"
+
+    def _compute_loc_accuracy(self):
+        for rec in self:
+            history = self.env["stock.inventory"].search([("state", "=", "done")])
+            history = history.filtered(
+                lambda x: rec.id in x.mapped("location_ids.id")
+            ).sorted(key=lambda r: r.write_date, reverse=True)
+            if history:
+                wh = rec.get_warehouse()
+                if (
+                    wh.counts_for_accuracy_qty
+                    and len(history) > wh.counts_for_accuracy_qty
+                ):
+                    rec.loc_accuracy = mean(
+                        history[: wh.counts_for_accuracy_qty].mapped(
+                            "inventory_accuracy"
+                        )
+                    )
+                else:
+                    rec.loc_accuracy = mean(history.mapped("inventory_accuracy"))
+            else:
+                rec.loc_accuracy = 0.0
+
+    zero_confirmation_disabled = fields.Boolean(
+        string="Disable Zero Confirmations",
+        help="Define whether this location will trigger a zero-confirmation "
+        "validation when a rule for its warehouse is defined to perform "
+        "zero-confirmations.",
+    )
+    cycle_count_disabled = fields.Boolean(
+        string="Exclude from Cycle Count",
+        help="Define whether the location is going to be cycle counted.",
+    )
+    qty_variance_inventory_threshold = fields.Float(
+        string="Acceptable Inventory Quantity Variance Threshold"
+    )
+    loc_accuracy = fields.Float(
+        string="Inventory Accuracy", compute="_compute_loc_accuracy", digits=(3, 2)
```

Comment:
```suggestion
        string="Inventory Accuracy", compute="_compute_loc_accuracy", digits=(3, 2), default=0.00
```

Reply:
you can but this causes a CacheMiss error

From OCA migration guidelines:
- Check that all "compute" methods of non-stored computed fields assign a value in any case to the field, even if it is a falsy one. (https://github.com/odoo/odoo/pull/36743/commits/2e43bfc1c4b2f61e0459614f61f90a77dc3b7233).

- Computed stored fields will keep their previous value if not assigned during the compute method, so don't rely on any expected default value.